pkgs/by-name/si/sillytavern/package.nix

```diff
@@ -0,0 +1,123 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, makeWrapper
+, pkgs
+}:
+
+let
+wrapperScript = pkgs.writeShellScriptBin "sillytavern-wrapper" ''
```

Comment:
I'd like to go through the contents of this wrapper script in more detail before signing off on this PR. I will do so soon when I have a bit more time to dig into it.